src/test/java/org/jenkinsci/plugins/workflow/job/properties/QueryingMockTrigger.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
```

Comment:
Well, I actually am the one who wrote the code in this PR. @KostyaSha just opened it from my repo. =)